library/network_connections.py

```diff
@@ -1148,6 +1148,61 @@ def connection_create(self, connections, idx, connection_current=None):
                     s_ip4.add_route(rr)
                 else:
                     s_ip6.add_route(rr)
+            for r in ip["routing_rules"]:
+                routing_rule = NM.IPRoutingRule.new(r["family"])
+                if r["priority"] is not None:
```

Comment:
the priority is mandatory. `ArgValidator` should always ensure that it's present.